core/src/main/java/org/elasticsearch/index/query/BoolQueryBuilder.java

```diff
@@ -235,10 +235,10 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
         doXArrayContent("filter", filterClauses, builder, params);
         doXArrayContent("must_not", mustNotClauses, builder, params);
         doXArrayContent("should", shouldClauses, builder, params);
-        builder.field("disable_coord", disableCoord);
```

Comment:
Putting comment here for lack of beeing ablto put it at unchanged lines: just an idea, would it make sense to also introduce string constants for must/filter etc... above? I see that its not easy using them as parse fields in the current parser due to the "switch", but maybe we can change them to constant so they are all in one place, e.g. the builder?